### PR TITLE
provisioning: add instructions for IBM Cloud

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: modules/ROOT/pages/provisioning-ibmcloud.adoc
Comment (lucab) [inline modules/ROOT/pages/provisioning-ibmcloud.adoc:22]: Nit: I think `ibmcloud login -r ${REGION} --apikey @my-key.json` is equivalent here and also covers the API-login step (but please double-check that it also affects commands further below).

File: modules/ROOT/pages/provisioning-ibmcloud.adoc
Comment (lucab) [inline modules/ROOT/pages/provisioning-ibmcloud.adoc:9]: s/etc../etc./ ?

File: modules/ROOT/pages/provisioning-ibmcloud.adoc
Comment (lucab) [inline modules/ROOT/pages/provisioning-ibmcloud.adoc:66]: I think something went wrong with this sentence, possibly some copy-paste mixup?

File: modules/ROOT/pages/provisioning-ibmcloud.adoc
Comment (dustymabe) [reply modules/ROOT/pages/provisioning-ibmcloud.adoc:22]: I was trying to explicitly avoid giving the user the `ibmcloud login` command to use because they could be using an API key or they could be using a username/password. They probably do set the region when they log in, but having this as a separate command does enable us to set the `$REGION` environment varialbe, which we do use later.

File: modules/ROOT/pages/provisioning-ibmcloud.adoc
Comment (dustymabe) [reply modules/ROOT/pages/provisioning-ibmcloud.adoc:66]: indeed :spaghetti:

will fix

Comment (dustymabe) [reply modules/ROOT/pages/provisioning-ibmcloud.adoc]: New wording in last push. Hopefully that is better.

**Review by lucab — APPROVED** [review]

**Comment by lucab** [comment]
I think @dustymabe wants to hold this for a bit and synchronize with the rest of the work at https://github.com/coreos/fedora-coreos-tracker/issues/277.
This LGTM as is (but I haven't tried a full provisioning flow right now, I stopped midway due to lack of ACL/time), I'll let him merge when he prefers.

**Review by travier — APPROVED** [review]
Not tested but LGTM.

**Comment by dustymabe** [comment]
Thanks for the reviews. I'll merge after the next round of releases (which should all contain IBM Cloud images IIUC).

**Pull request merged**
